frontends/ol-components/src/components/LearningResourceCard/LearningResourceListCard.tsx

```diff
@@ -0,0 +1,323 @@
+  resource.platform?.code === PlatformEnum.Ocw
+
+const getStartDate = (resource: LearningResource) => {
+  let startDate = resource.next_start_date
+
+  if (!startDate) {
+    const bestRun = findBestRun(resource.runs ?? [])
+
+    if (isOcw(resource) && bestRun?.semester && bestRun?.year) {
+      return `${bestRun?.semester} ${bestRun?.year}`
+    }
+    startDate = bestRun?.start_date
+  }
+
+  if (!startDate) return null
+
+  return formatDate(startDate, "MMMM DD, YYYY")
+}
+
+const StartDate: React.FC<{ resource: LearningResource }> = ({ resource }) => {
+  const startDate = getStartDate(resource)
+
+  if (!startDate) return null
+
+  const label = isOcw(resource) ? "As taught in:" : "Starts:"
+
+  return (
+    <div>
+      {label} <span>{startDate}</span>
+    </div>
+  )
+}
+
+const Format = ({ resource }: { resource: LearningResource }) => {
+  const format = resource.learning_format?.[0]?.name
+  if (!format) return null
+  return (
+    <div>
+      Format: <span>{format}</span>
+    </div>
+  )
+}
+
+const Loading = styled.div<{ mobile?: boolean }>`
+  display: flex;
+  padding: 24px;
+  justify-content: space-between;
+
+  > div {
+    width: calc(100% - 236px);
+  }
+
+  > span {
+    flex-grow: 0;
+    margin-left: auto;
+  }
+  ${({ mobile }) =>
+    mobile
+      ? `
+    padding: 0px;
+    > div {
+      padding: 12px;
+    }`
+      : ""}
+`
+
+const LoadingView = ({ isMobile }: { isMobile: boolean }) => {
+  const { width, height } = IMAGE_SIZES[isMobile ? "mobile" : "desktop"]
+  return (
+    <Loading mobile={isMobile}>
+      <div>
+        <Skeleton
+          variant="text"
+          width="15%"
+          style={{ marginBottom: isMobile ? 4 : 10 }}
+        />
+        <Skeleton
+          variant="text"
+          width="75%"
+          style={{ marginBottom: isMobile ? 16 : 51 }}
+        />
+        <Skeleton variant="text" width="20%" />
+      </div>
+      <Skeleton
+        variant="rectangular"
+        width={width}
+        height={height}
+        style={{ borderRadius: 4 }}
+      />
+    </Loading>
+  )
+}
+
+interface LearningResourceListCardProps {
+  isLoading?: boolean
+  resource?: LearningResource | null
+  className?: string
+  href?: string
+  onAddToLearningPathClick?: ResourceIdCallback | null
+  onAddToUserListClick?: ResourceIdCallback | null
+  editMenu?: React.ReactNode | null
+}
+
+const LearningResourceListCard: React.FC<LearningResourceListCardProps> = ({
+  isLoading,
+  resource,
+  className,
+  href,
+  onAddToLearningPathClick,
+  onAddToUserListClick,
+  editMenu,
+}) => {
+  const isMobile = !useMuiBreakpointAtLeast("md")
+
+  if (isLoading) {
+    return (
+      <ListCard className={className}>
+        <ListCard.Content>
+          <LoadingView isMobile={isMobile} />
+        </ListCard.Content>
+      </ListCard>
+    )
+  }
+  if (!resource) {
+    return null
+  }
+  return (
+    <ListCard href={href || `?resource=${resource.id}`} className={className}>
```

Comment:
**Requested Change:** This `href` does not behave correctly for the search page—it replaces the search query parameters rather than augmenting them.

This does work
```
const { search } = useLocation()
// then later

<ListCard
  href={href || `${search}${search ? "&" : "?"}resource=${resource.id}`}
  className={className}
>
```

**HOEVER:** It feels strange to me that `ol-components` is aware of what route the resource drawer is at. (Route-awareness belongs to page-components).

IMO, let's just specify the href via the parent. (Although, I think having a page-component with the button actions and href builtin would be reasonable. That's what we used to do, with LearningResourceCardTemplate and LearningResourceCard. Don't love the "template" language, though. )

Reply:
Yes, intended as a default, though I agree it should be controlled by the caller (and total miss on replacing the whole search!).

I've added a LearningResourceDrawer hook to provide the href.